xwiki-rendering-syntaxes/xwiki-rendering-syntax-xhtml/src/main/java/org/xwiki/rendering/internal/parser/xhtml/wikimodel/XWikiWikiModelHandler.java

```diff
@@ -85,4 +85,11 @@
      */
     @Unstable
     String NON_GENERATED_CONTENT_STACK = "nonGeneratedContentStack";
+
+    /**
+     * Stack parameter which record the name of the parameter the current content is dedicated to.
+     * @since 11.1RC1
+     */
+    @Unstable
```

Comment:
This is not an APi since it's an internal package so Unstable annotation is n not needed. I did not notice one was associated with NON_GENERATED_CONTENT_STACK.

Reply:
Last changes were pretty huge, no surprise that some things were missing ;) I'll remove them both.